spatialos-sdk/src/worker/connection.rs

```diff
@@ -1,18 +1,21 @@
 use crate::ptr::MutPtr;
-use crate::worker::commands::*;
-use crate::worker::component::internal::ComponentUpdate;
-use crate::worker::component::{self, Component};
-use crate::worker::entity::Entity;
-use crate::worker::internal::utils::cstr_to_string;
-use crate::worker::locator::*;
-use crate::worker::metrics::Metrics;
-use crate::worker::op::OpList;
-use crate::worker::parameters::ConnectionParameters;
-use crate::worker::{EntityId, InterestOverride, LogLevel, RequestId};
+use crate::worker::{
+    commands::*,
+    component::{self, internal::ComponentUpdate, Component},
+    entity::Entity,
+    internal::utils::cstr_to_string,
+    locator::*,
+    metrics::Metrics,
+    op::OpList,
+    parameters::ConnectionParameters,
+    {EntityId, InterestOverride, LogLevel, RequestId},
+};
```

Comment:
@jamiebrynes7 This is a good example of a place where I think the nested formatting style achieves the type of import grouping you're looking for. If this seems reasonable to you, I'll start using this going forward :)